modules/reindex/src/yamlRestTest/resources/rest-api-spec/test/reindex/20_validation.yml

```diff
@@ -308,7 +308,7 @@
 ---
 "unwhitelisted remote host fails":
```

Comment:
Can this test description be changed?

Reply:
Good catch! Thank you. This one is missed in PR #2178, I changed it in this PR.